dashboard/src/features/app/components/Navigation/Navigation.jsx

```diff
@@ -56,7 +56,7 @@ class Navigation extends React.Component {
 
         <ul className={styles.navigation}>
           <li className={styles.navigationTitle}>services</li>
-          {!this.props.production &&
+          {!this.props.production && this.props.mockhsm &&
```

Comment:
Should we get rid of the production flag here? I would think now that there's a mockhsm flag that's the only thing that's relevant. Thoughts?